code/modules/clothing/suits/jobs.dm

```diff
@@ -155,7 +155,17 @@
 	name = "surgical apron"
 	desc = "A sterile blue surgical apron."
 	icon_state = "surgical"
-	allowed = list(/obj/item/scalpel, /obj/item/surgical_drapes, /obj/item/cautery, /obj/item/hemostat, /obj/item/retractor)
```

Comment:
would you mind readding these items (and maybe some of the other surgery tools) to the allowed list of surgical aprons? idk, maybe some doctor out there likes keeping their (laser) scalpel in their suit storage slot or something.

Reply:
Re-added surgical tools to the list, added advanced tools as well